Assignment/traning4/src/task2/components/searchCountry/searchCountry.js

```diff
@@ -0,0 +1,56 @@
+import React, { useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import axios from 'axios';
+import './searchCountry.css';
+
+
+
+SearchCountry.propTypes = {
+    onSubmit: PropTypes.func,
+}
+
+SearchCountry.defaultProps = {
+    onSubmit: null,
+}
+
+
+function SearchCountry(props) {
+
+    const {onSubmit} = props;
+    const typingTimeoutRef = useRef(null);
+    const [search, setSearch] = useState('');
+
+
+    const handleChange = (e) =>
```

Comment:
@lehuy1012 handle change created every you change the search, I don't think your debounce work is correct because it makes create much setTimeout run parallel, please use the lodash debounce and refactoring the code

Reply:
fixed